tests/eko/io/test_legacy.py

```diff
@@ -1,19 +1,88 @@
-import pytest
+import pathlib
+from math import nan
+import yaml
+from banana import toy
 
-from eko.io import legacy
+import eko
 
+import numpy as np
+from eko.interpolation import make_grid
+from ekobox.evol_pdf import evolve_pdfs
+import lhapdf
+from ekobox.apply import apply_pdf
+import numpy.testing
+from numpy.testing import assert_almost_equal
+import ekomark.benchmark.runner
 
-def test_op5to4():
-    with pytest.raises(RuntimeError, match="not found"):
-        legacy.op5to4([], {})
+TEST_DATA_DIR = pathlib.Path(__file__).parents[2] / "data"   # directory of the EKO object
+pdf = toy.mkPDF("",0)
 
-    mu2 = 1.959
-    op = 29348.2342
-    err = 54225.24
-    op4s = legacy.op5to4([mu2], {legacy.OPERATOR: [op], legacy.ERROR: [err]})
-    assert mu2 in op4s
-    assert op4s[mu2].operator == op
-    assert op4s[mu2].error == err
+x_grid = [1e-7, 1e-6, 1e-5, 1e-4, 1e-3, 1e-2, 0.1, 0.3, 0.5, 0.7, 0.9]
+Q2 = 10000
```

Comment:
```suggestion
EP = (10000., 4)
```
and use that consistently. (recall that conventionally EvolutionPoint -> ep)

Reply:
note the `.` - this makes a difference!